### PR TITLE
remove(docs): remove sending telemetry feature

### PR DESCRIPTION
**Pull request by yanavasileva**
https://github.com/camunda/camunda-bpm-platform/issues/4483

### CODE REVIEW

File: content/update/minor/721-to-722/_index.md [hunk 1]
```diff
@@ -203,3 +217,41 @@ It is possible to keep using Camunda Platform with a lower version of Groovy if
 ### If using Embedded Engine:
 
 - You can keep using your Groovy version without any extra effort.
+
+# Sending telemetry feature removed
+
+Sending telemetry data has been introduced in Camunda `7.14.0+` (and `7.13.7`, `7.12.12`, `7.11.19`)
+ and removed in Camunda `7.22.0`. The public API is marked as deprecated and emptied out.
+ The telemetry reporter and scheduled timer and all related process engine configuration properties are removed.
+ The [diagnostic data][] is still being collected but not send by any mean.
```
Comment (joaquinfelici) [inline content/update/minor/721-to-722/_index.md:226]: Seems there's a missing link here.

Comment (joaquinfelici) [inline content/update/minor/721-to-722/_index.md:226]:
```suggestion
 The [diagnostic data][] is still being collected but not sent by any mean.
```

File: content/update/minor/721-to-722/_index.md [hunk 2]
```diff
@@ -203,3 +217,41 @@ It is possible to keep using Camunda Platform with a lower version of Groovy if
...
+  You can decide to share it with Camunda if requested in tickets or use it for your diagnostic purposes.
+
+In previous Camunda versions - `7.21.0+`, `7.20.8+`, `7.19.15+`, reporting telemetry data is disabled by default.
+
+[diagnostic data]: {{< ref "/user-guide/process-engine/diagnostics-data.md" >}}
+
+## Configuration properties removed
+
+To clean up and refactor our source code, tte following process engine configuration properties have been remove.
```
Comment (joaquinfelici) [inline content/update/minor/721-to-722/_index.md:235]:
```suggestion
To clean up and refactor our source code, tte following process engine configuration properties have been removed.
```

File: content/update/minor/721-to-722/_index.md [hunk 3]
```diff
@@ -203,3 +217,41 @@ It is possible to keep using Camunda Platform with a lower version of Groovy if
...
+Please remove all of the occurrences of those properties, no matter of the setup that you are using
```
Comment (joaquinfelici) [inline content/update/minor/721-to-722/_index.md:236]:
```suggestion
Please remove all of the occurrences of those properties, regardless of the setup that you are using
```

File: content/update/minor/721-to-722/_index.md [hunk 4]
```diff
@@ -203,3 +217,41 @@ It is possible to keep using Camunda Platform with a lower version of Groovy if
...
+* `telemetryReporterActivate `
+* `telemetryReportingPeriod  `
+* `telemetryReporterActivate `
+* `telemetryRequestRetries   `
+* `telemetryRequestTimeout   `
+
+## Public API deprecation
+
+The public API for configuring telemetry and fetching the telemetry configuration have been emptied out and marked as deprecated. 
+We recommend to remove the usage of the following API as the endpoints no longer do anything and is unnecessary.
```
Comment (joaquinfelici) [inline content/update/minor/721-to-722/_index.md:250]:
```suggestion
We recommend to remove the usage of the following API as the endpoints no longer do anything and their usage is unnecessary.
```

**Review by joaquinfelici — CHANGES_REQUESTED** [review]
Looks good! Added some suggestions.

File: content/update/minor/721-to-722/_index.md (hunk 1 above)
Comment (yanavasileva) [reply content/update/minor/721-to-722/_index.md]: Discussed. The link is declared 4 rows below.

**Review by joaquinfelici — APPROVED** [review]
Looks good thanks for applying suggestions 👍

**Pull request merged**